cpp/oneapi/dal/algo/linear_kernel/BUILD

```diff
@@ -17,17 +17,33 @@ dal_module(
 
```

Comment:
Is this file actually related to PCA?

Reply:
I've added support for DPC++ tests in Bazel to be able to run DPC++ PCA tests and had to fix existing rules/tests.